.github/workflows/tester.yml

```diff
@@ -25,21 +25,34 @@ jobs:
       - name: Install Dependencies
         run: npm install
       - name: Test
+        run: npm run test
+        env:
+          CI: true
+  coverage:
```

Comment:
in previous draft, this job is integrated into `tester` job with `if: ${{ matrix.os == 'ubuntu-latest' && matrix.node-version == '12.x' }}`, but then `tester` would have one extra mocha execution which defeats the purpose.